src/pages/connect/github.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { handleAuthentication } from '../../services/auth'
+
+const Github = () => {
+  const url = process.env.GATSBY_STRAPI_API_URL || 'http://localhost:8082'
+  const { jwt } = handleAuthentication(url, 'github')
```

Comment:
I don't think you need to destructure `{jwt}`? it's not used anywhere

Reply:
Yeah, actually no need to return anything from `handleAuthentication`